db/version_edit_test.cc

```diff
@@ -309,6 +309,102 @@ TEST_F(VersionEditTest, BlobFileAdditionAndGarbage) {
   TestEncodeDecode(edit);
 }
 
+TEST_F(VersionEditTest, AddWalEncodeDecode) {
+  VersionEdit edit;
+  for (uint64_t log_number = 1; log_number <= 20; log_number++) {
+    edit.AddWal(log_number, WalMetadata(rand() % 100));
+  }
+  TestEncodeDecode(edit);
+}
+
+TEST_F(VersionEditTest, AddWalDebug) {
+  const int n = 2;
```

Comment:
Minor: this could be

```
constexpr size_t n = 2;
constexpr std::array<uint64_t, n> kLogNumbers{10, 20};
constexpr std::array<uint64_t, n> kBytes{100, 200};
```

(same below for deletes)